src/device-lister.js

```diff
@@ -137,16 +138,36 @@ export default class DeviceLister extends EventEmitter {
             });
     }
 
+    _onUsbDetach(usbDevice) {
+        debug(`Called _triggerReenumeration because of detached USB device VID/PID 0x${
+            usbDevice.deviceDescriptor.idVendor.toString(16).padStart(4, '0')}/0x${
+            usbDevice.deviceDescriptor.idProduct.toString(16).padStart(4, '0')}`);
```

Comment:
Could consider using one of the `hexpad4()` or `getDeviceId()` functions from util/usb.js.